boards/arm64/imx8mm_evk/imx8mm_evk_defconfig

```diff
@@ -0,0 +1,39 @@
+# SPDX-License-Identifier: Apache-2.0
+
+# Platform Configuration
+CONFIG_SOC_SERIES_IMX8M_A53=y
+CONFIG_SOC_IMX8MM_A53=y
+CONFIG_BOARD_IMX8MM_EVK_A53=y
+CONFIG_ARM_ARCH_TIMER=y
+
+# Zephyr Kernel Configuration
+CONFIG_XIP=n
+CONFIG_ARM64_VA_BITS_48=y
+CONFIG_ARM64_PA_BITS_48=y
+
+# Serial Drivers
+CONFIG_SERIAL=n
+CONFIG_UART_INTERRUPT_DRIVEN=n
+
+# Enable Console
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=n
+
+CONFIG_RAM_CONSOLE=y
```

Comment:
From the DTS and the SOC MMU regions, it appears that this enablement expects the A53 to output data via the UART console. Should the default board `defconfig` enable the UART console, and the `jailhouse` target use the `RAM_CONSOLE`?

Reply:
https://github.com/zephyrproject-rtos/zephyr/pull/32694 This is the original PR, because github PR not auto updated after I update my branch, new PR created. If you have intrested, you could have a look of all the discussions there. As I raised above, MCUX HAL is not ready, I could not use hardware UART now, so I use ram console. The other PR is adding jailhouse debug console, after that PR in, I will switch to use jailhouse debug console. For default defconfig, still needs to wait MCUX HAL ARM64 ready.